Data Hub Administration changes (#3578)
mendix/docs: content/data-hub/general/data-hub-admin-functions.md

Curators and the Data Hub Admin can manage the custom owner list under the **Owner Management** tab.

{{% alert type="info" %}}
Curators will only see  **Owner Management** under the **Administration** tab.
{{% /alert %}}

From the **Owner Management** tab, you can do the following:

* Add owners – the **Add Owners** button allows you to add new owners by filling in their name and email. New owners will be listed as the **Business** or **Technical Owners** for registered assets.

* Edit the details of the listed owners by clicking the edit pencil icon.
* Delete names from the list by clicking the **x** icon and confirming your choice.

	{{% alert type="info" %}}If a custom owner is removed from the list, they will also be removed from any registered assets where they were set as the owner. This means that the asset will not have a contact.{{% /alert %}}

New custom owners can also be added when assets are being curated as described in [Changing the Technical and Business Owners of an App](/data-hub/data-hub-catalog/curate#custom-owner).

## 4 Managing External Access {#external-access}

> **External Access** allows Data Hub Admins to invite members (external users) from outside the organization. External users can search for published data sources or use them in apps. External users cannot register or curate content unless they are an owner of the data source.

Reviewer: Can we have multiple Data Hub admins or not? In the introduction of this document, we did mention "Every organization has one Mendix Data Hub Administrator."

Reviewer: I'm fine with plural or singular. I'm more concerned about consistency.

Author: Is having at least one Admin obligatory? Then I can write: "Every organization has at least one Mendix Data Hub Administrator." 
or I can say:
"Every organization can have one or several Mendix Data Hub Administrator."

Reviewer 2: I checked, organization can have no admins at all. But the current phrasing `Every organization can have one or several Mendix Data Hub Administrator` seems ok.

![External Access](attachments/data-hub-admin/ExternalAccessOverview.PNG)

Data Hub Admin can manage the external user list in the **External Access** screen. 

To invite new users, do the following:

1. In the Data Hub Catalog, go to the **Administration** tab > **External Access** screen.

2. Click the **Invite Users** button.

3. In the **Invite External Users** dialog box, fill in the email addresses of the users you would like to invite as external users and click **Send Invitation**.     ![External Access](attachments/data-hub-admin/InviteExternalUser.PNG)

    {{% alert type="info" %}}The users you are inviting should have a Mendix account. {{% /alert %}}

> 4. Users receive the invitation, log into the Data Hub Catalog, and accept the invitation.

Reviewer: Having Mendix account doesn't mean they have Data Hub account, right?
Do users use their Mendix credentials to log into Data Hub Catalog or not?
This part isn't super clear for me.

Reviewer 2: > Having Mendix account doesn't mean they have Data Hub account, right?

It is. Just that company has to have a license for Data Hub for their users to access the Catalog. Otherwise they will land on 403 page.

> Do users use their Mendix credentials to log into Data Hub Catalog or not?

All Mendix applications are using single MxID3 authentication system. This means that users get access to entire platform with single identity and profile.

Reviewer: Ok. Thanks for the explanation! I wonder if it helps to mention explicitly "log into the Data Hub Catalog with your Mendix credentials..."? What do you think?

Reviewer 2: Can be, good clarification. @MariaShaposhnikova what do you think?

Author: I've added it to the document.

>     {{% alert type="info" %}}A user can only work in one organization at a time. To work actively in an organization as an external user, they need to select an **Active company** option from the Catalog homepage.{{% /alert %}}

Reviewer: I feel the wording of this sentence can be misleading. Working in one or more organization is different from working in multiple data hub.
One suggestion: A user can only work in **one Data Hub** at a time.
I'm not so sure what would be 100% correct. Any thoughts?

Reviewer 2: I'd suggest something like "A user can access resources from one organization at a time." Later it explain the need to switch active company.

Reviewer: Sounds good! Thanks!

>     {{% alert type="info" %}}A user can only work in one organization at a time. To work actively in an organization as an external user, they need to select an **Active company** option from the Catalog homepage.{{% /alert %}}

Reviewer: Users won't be able to find **Active company** literally in Catalog home page. It will be helpful to add a screenshot to point out that there is a small component at the top-right corner of the screen.

Author: Banani added the screenshot.

The Data Hub Admin can add or remove the curator role by following these steps:

> 1. From the **Data Hub** screen, click the **Administration** tab:

Author: I'll say "from the Data Hub Catalog".

	![Administration](attachments/data-hub-admin/administration.png)

2. The Data Hub Admin will see the **Curator Management** tab displaying  the list of Data Hub curators for the organization. To assign a curator role to a Mendix user, click **Add Curator**.

>     {{% alert type="info" %}}A user with curator rights can access all assets registered in the Catalog. This also includes those that are set to **Non-discoverable**. Curators can also change the information that is registered for assets that are owned by other users. {{% /alert %}}

Author: 1. Added.
2. Changed to "metadata".

> 3. If you want to remove the curator rights for a user, check the box against the name and confirm this by clicking **Remove Curator**.

Author: I have no access, so I can't provide any screenshots.

> 3. If you want to remove the curator rights for a user, check the box against the name and confirm this by clicking **Remove Curator**.

Reviewer 3: New screenshot of **Remove Curator** functionality  is added (made from test environment ).

{{% alert type="info" %}}This list shows all Mendix users for your organizations. It does not include non-Mendix users that are added as *Custom Owners*. For more information, see [Managing Custom Owners](#customowners).{{% /alert %}}

## 3 Managing Custom Owners {#customowners}

Custom owners are owners that have been added as the contact for a registered application. They may be added during the application [curation](/data-hub/data-hub-catalog/curate#custom-owner) or have been specified during app registration.

They are  the contacts for registered assets. Adding a custom owner does not give them access rights to the Catalog. Custom owners are displayed in the drop-down lists by an avatar that only displays their initials (Mendix users have their personalized avatar displayed).

Curators and the Data Hub Admin can manage the custom owner list under the **Owner Management** tab.

{{% alert type="info" %}}
Curators will only see  **Owner Management** under the **Administration** tab.
{{% /alert %}}

From the **Owner Management** tab, you can do the following:

> * Add owners – the **Add Owners** button allows you to add new owners by filling in their name and email. New owner will be listed when **Business** or **Technical Owners** are curated for registered assets.

Reviewer 4: `New owner will be listed when **Business** or **Technical Owners** are curated for registered assets.` Do you mean when I go look at the endpoint, I'll see the new owner? Or is curation an extra step?

```suggestion
* Add owners – the **Add Owners** button allows you to add new owners by filling in their name and email. New owners will be listed as the **Business** or **Technical Owners** for registered assets.
```

Author: I only changed description to align with our style, I did not change any technical details. If this is not clear from a technical perspective, please change it or tell me what I should change it to.

> 	{{% alert type="info" %}}Custom owners in this list are not Mendix platform users but serve as contact for the registered assets.  They are not able to login to the Mendix Platform or curate registered assets in the Catalog. {{% /alert %}}

Reviewer 4: You said this on line 54.

Author: I'll add "They are not able to login to the Mendix Platform or curate registered assets in the Catalog" to line 54 and delete this note.

* Edit the details of the listed owners by clicking the edit pencil icon.
* Delete names from the list by clicking the **x** icon and confirming your choice.

	{{% alert type="info" %}}If a custom owner is removed from the list, they will also be removed from any registered assets where they were set as the owner. This means that the asset will not have a contact.{{% /alert %}}

New custom owners can also be added when assets are being curated as described in [Changing the Technical and Business Owners of an App](/data-hub/data-hub-catalog/curate#custom-owner).

## 4 Managing External Participants {#external-participants}

> **External Access** allows Data Hub Admin to invite members **(External participant)** from outside the organization. **External participant** has certain access such as searching published data sources or using them in apps. However, external participants cannot register or curate content of the company unless they are the owner of the data source.

Reviewer 4: I thought Reinout made up the name `External participant`, is that what we want customers to call it too? 

```suggestion
**External Access** allows Data Hub Admins to invite members **(External participants)** from outside the organization. **External participants** have can search for published data sources and using them in apps. External participants cannot register or curate content unless they are an owner of the data source.
```

Reviewer 2: In the emails to the customers Georg used term `External user` as in `Last but not least, anyone external user invited to your tenant cant register or curate content, unless they are the owner.`.

Author: I could only review language and style, I thought external participants is what users see in the UI at some point. If not, shall we call them "External access users"?

Author: I'll change "external participants" to "external users" and will also impliment Elizabeth's changes.

> ![External Access](attachments/data-hub-admin/ExternalAccessOverview.PNG)

Reviewer 4: This screenshot shows the `Curator Management` and `Owner Management` on the left navigation. An earlier screenshot shows them at the top of my screen. Should I expect them in two different spots?

Author: I have no access to it and these screenshots were already there. Which of the two is the correct one?

Reviewer 3: These are old screenshots , Will replace the Curator management and Owner Management with new screenshots

Data Hub Admin can manage the external participants list in the **External Access** screen. 

To invite new users, do the following:

1. From the **Data Hub** screen, go to the **Administration** tab > **External Access** screen.

2. Click the **Invite Users** button.

3. In the **Invite External Users** dialog box, fill in the email addresses of the users you would like to invite as external participants and click **Send Invitation**.     ![External Access](attachments/data-hub-admin/InviteExternalUser.PNG)

>     {{% alert type="info" %}}The users you are inviting should have a Mendix account. {{% /alert %}}

Reviewer 4: Tell me what happens if they don't have a Mendix account.

Author: @BananiMendix, please add this information.

Reviewer: > Tell me what happens if they don't have a Mendix account.

I have the same question. I thought we can send invitation to a non-Mendix user.

Reviewer 2: > I thought we can send invitation to a non-Mendix user.

This is postponed for now as inviting non-Mendix users has some technical complexity around it. Initially we invited only existing Data Hub users, then added Data Hub account pre-provisioning on invitation.

Reviewer 3: Rephrasing it to following ,with a screenshot "You can invite only users with a mendix account.Invitations will not be sent to the emails which are not associated with a mendix account."
![nonMx_Invitee](https://user-images.githubusercontent.com/88437796/128725299-735583fd-504d-4eeb-881c-c26a459a5389.PNG)

Reviewer: Hi Banani, thanks for sharing! I think in this case, it will be quite important to actually show which users (the email addresses) are not Mendix members. Is that feasible?

Author: I've deleted the screenshot because it actually does not add much value to what is already written in the note. We also do not add screenshots to notes in general, this is not the best practice.
As for the email addresses, please note we cannot show real data examples.

> 4. Users receive the invitation, log into Data Hub, and accept the invitation.

Reviewer 4: ```suggestion
4. Users receive the invitation, log into Data Hub Catalog, and accept the invitation.
```
